### PR TITLE
feat(core): expose additional_browser_args to window config (fix: #5757)

### PR DESCRIPTION
**Pull request by noamzaks**
### What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [x] No

### Checklist
- [x] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [x] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
- [ ] I have added a convincing reason for adding this feature, if necessary

### Other information
Could probably help various use cases, including mine in #5757.

[Discord thread](https://discord.com/channels/616186924390023171/1048227579066331216).

### CODE REVIEW

File: core/tauri-runtime/src/webview.rs [hunk 1]
```diff
@@ -88,6 +90,13 @@ impl WebviewAttributes {
     self.accept_first_mouse = accept;
     self
   }
+
+  /// Sets additional browser arguments on **Windows**.
```
Comment (amrbashir) [inline core/tauri-runtime/src/webview.rs]:
```suggestion
  /// Sets additional browser arguments. **Windows Only**
```

File: core/tauri-utils/src/config.rs [hunk 2]
```diff
@@ -881,6 +881,9 @@ pub struct WindowConfig {
   /// [tabbing identifier]: <https://developer.apple.com/documentation/appkit/nswindow/1644704-tabbingidentifier>
   #[serde(default, alias = "tabbing-identifier")]
   pub tabbing_identifier: Option<String>,
+  /// Defines additional browser arguments on Windows.
```
Comment (amrbashir) [inline core/tauri-utils/src/config.rs]:
```suggestion
  /// Defines additional browser arguments. **Windows Only**
```

File: core/tauri/src/window.rs [hunk 3]
```diff
@@ -526,6 +526,13 @@ impl<'a, R: Runtime> WindowBuilder<'a, R> {
     self
   }
 
+  /// Set additional arguments for the webview on **Windows**.
```
Comment (amrbashir) [inline core/tauri/src/window.rs]:
```suggestion
  /// Set additional arguments for the webview. **Windows Only**
```

File: tooling/api/src/window.ts [hunk 4]
```diff
@@ -2060,6 +2060,10 @@ interface WindowOptions {
    * The user agent for the webview.
    */
   userAgent?: string
+  /**
+   * Additional arguments for the webview on Windows.
```
Comment (amrbashir) [inline tooling/api/src/window.ts]:
```suggestion
   * Additional arguments for the webview. **Windows Only**
```

**Review by amrbashir — CHANGES_REQUESTED** [review]
Thank you, just a few nit-picks and should be good to go

**Review by amrbashir — DISMISSED** [review]

**Comment by FabianLars** [comment]
I think we should document the default value like we do in wry: https://docs.rs/wry/latest/x86_64-pc-windows-msvc/wry/webview/trait.WebViewBuilderExtWindows.html#tymethod.with_additional_browser_args

**Review by lucasfernog — APPROVED** [review]

**Pull request merged**